app.js

```diff
@@ -1,11 +1,65 @@
-/* Imports */
-
+/* Imports and Get Dom Elements */
+const kanyeButtonEl = document.getElementById('kanye-button');
+const peteButtonEl = document.getElementById('pete-button');
+const doctorButtonEl = document.getElementById('doctor-button');
 /* Get DOM Elements */
 
+const kanyeContainer = document.getElementById('kanye-container');
+const peteContainer = document.getElementById('pete-container');
+const doctorContainer = document.getElementById('doctor-container');
+
+const correctGuessesEl = document.getElementById('wins');
+console.log(correctGuessesEl);
```

Comment:
In the future - you can delete or comment out your console.log() once you've verified the funciontality.